### PR TITLE
Add filter for the last messages forum widget

### PR DESCRIPTION
**Pull request by Suri4**
Add the possibility to configure the last messages forum widget.
In the form, adding of a checkbox which display the last messages of the connected user.
Return the five last messages instead of three.

### CODE REVIEW

File: Controller/WidgetController.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+
+class WidgetController extends Controller
+{
+    /**
+     * @Route("/widget/listmessages/{id}/config", name="claroline_forum_last_message_widget_configure", requirements={"id" = "\d+"})
+     * @Method("POST")
+     */
+    public function updateWidgetBlogList(Request $request, WidgetInstance $widgetInstance)
```
Comment (ngodfraind) [inline Controller/WidgetController.php]: The name is confusing. Is it's not a blog.

**Comment by Suri4** [comment]
Don't merge now, I find a bug ...

File: Controller/WidgetController.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            $entityManager = $this->get('doctrine.orm.entity_manager');
+
+            $entityManager->persist($lastMessageWidgetConfig);
+            $entityManager->flush();
+
+            return new Response('', Response::HTTP_NO_CONTENT);
+        }
+        else {
+
+//            var_dump($form->getData());
```
Comment (ngodfraind) [inline Controller/WidgetController.php]: The else condition is empty with var dumps and comments. It should be cleaned.

**Comment by Suri4** [comment]
You can merge now

**Pull request merged**